multiview/embed/omnibus.py

```diff
@@ -0,0 +1,160 @@
+"""
+omnibus.py
+==========
+Omnibus embedding for multiview dimensionality reduction.
+Code from the https://github.com/neurodata/graspy package,
+reproduced and shared with permission.
+"""
+
+from .base import BaseEmbed
+from ..utils.utils import check_Xs
+
+import numpy as np
+from graspy.embed import OmnibusEmbed
+from sklearn.metrics import pairwise_distances
+from sklearn.preprocessing import normalize
+
+
+class Omnibus(BaseEmbed):
+    """
+    Omnibus computes the pairwise distances for each view. Each
+    of these matrices is a n x n dissimilarity matrix where n is the number
+    of rows in each view. Omnibus embedding
+    (https://graspy.neurodata.io/reference/embed.html#multiple-graph-embedding)
+    is then performed over the dissimilarity matrices and the
+    computed embeddings are returned.
+
+    Parameters
+    ----------
+    n_components : strictly positive int (default = 2)
+        Desired dimensionality of output embeddings. See graspy docs for
+        additional details.
+
+    distance_metric : string (default = 'euclidean')
```

Comment:
are the other acceptable strings the typical ones for sklearn? if so, just mention that here

Reply:
Done!